.github/workflows/t2_reviews.yaml

```diff
@@ -0,0 +1,40 @@
+# Github Action to Automatically assign T2 Reviewers to framework touched PRs
+name: Auto assign T2 Reviewers
+
+on:
+  pull_request:
+    paths: ['.github/**']
+
+env:
+  number: ${{ github.event.number }}
+  reviewers: 'robottelo-tier-2-reviewers'
+  labels: 'Framework Changes'
+  GITHUB_TOKEN: ${{ secrets.CHERRYPICK_PAT }}
+
+jobs:
+  assign-reviewers-and-labels:
+    name: Assign T2 Reviewers and Labels to the PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add Labels to the PR
+        run:
+          echo "Adding applicable lables to the PR"
+          curl -L \
+          -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer $GITHUB_TOKEN"\
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          https://api.github.com/repos/jyejare/robottelo/issues/$number/labels \
+          -d '{"labels":[$labels]}'
+
+      - name: Assign T2 Reviewers Team to the PR
+        run:
+          echo "Assigning T2 Reviewers to the PR $number"
+          curl -L \
+          -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer $GITHUB_TOKEN"\
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          https://api.github.com/repos/jyejare/robottelo/pulls/$number/requested_reviewers \
```

Comment:
Instead of running this API's endpoints to ask for reviews, can you try exploring this [auto-request-review](https://github.com/marketplace/actions/auto-request-review) action?

Reply:
if you don't want to maintain a separate yml config for it, there's [review-request-action action](https://github.com/marketplace/actions/review-request-action) for it too, but it doesn't seem to be maintained since long

Reply:
I didnt just want to assign reviews but also add the labels and I dont want to use 2 different GHAs for simple requests, hence I choose to use a simple curl request which is always pure.

Not only this GHA but I dont find any GHA in the market which will do both for me.

Reply:
Well, so this are two different operations, you could use two different actions for it, but there isn't any single action which does both. And this isn't matter of having it **pure** or run curl commands, actions provide different features and provides flexibility to run actions in GHA.

Reply:
Yeah ofcourse GHA provides more options but those are only useful when you need them.

Since our requirement is minimum I stayed at lowest necessary thing. We can switch to those GHAs anytime when we feel we should and need.

Also I am bit concern on how much secure does these GHAs are and looking for some mechanism to scan these GHAs for their genuine certificates.